src/index.html

```diff
@@ -9,6 +12,56 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <a class="home_page_link" href="#home">
+        <img
+        class="logo_img--link-to-home-page"
+        src="images/logo.png"
+        alt="Logotype of MOYO"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
Follow naming convention `block__element--modifier`, check [here](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem)
in your case tag should have a class `nav__list`